### ISSUE TITLE
Converting encoding

### ISSUE THREAD

**Issue by ThotoBot** [notepad-plus-plus/notepad-plus-plus#5568]
### Description of the Issue

Documents in e.g. ANSI Encoding shall be converted to e.g. UTF-8
Feature deletes the content of the document instead of converting. 

### Steps to Reproduce the Issue

1. load document in e.g. ANSI encoding
2. Use Menue to convert to e.g. UTF-8

### Expected Behavior

Document shall be converted to the selceted encoding. 

### Actual Behavior

feature deletes the content of the document. 

### Debug Information

Notepad++ v7.6.6   (32-bit)
Build time : Apr  3 2019 - 23:49:50
Path : C:\PApps\PortableApps\Notepad++Portable\App\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll 

I think the Issue was also present in version 7.6.4.

**Comment by MetaChuh** [comment]
@ThotoBot 

thank you for your debug information.

**important note:** make sure you are **not** using the notepad++ 7.6.6 package from portableapps.com, as it is currently reported as broken.
please only download notepad++ from our [official download pages](https://notepad-plus-plus.org/download/). both installer and portable zip versions are officially available at this download location only.

i can not reproduce a deletion of the content of a document when converting an ansi file to utf-8 on official portable releases of notepad++ 7.6.6.

* please download the notepad++ 7.6.6. 64 bit portable version from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.bin.x64.zip).

* extract `npp.7.6.6.bin.x64.zip` to e.g. your desktop.

* **important note:** make sure to close all instances of notepad++ that might be running, before starting the portable version at the next step, to be sure you are using the portable version for this test.

* open the extracted `npp.7.6.6.bin.x64` folder and start this portable version by double-clicking on `notepad++.exe` inside this folder.

* please retry ansi to utf-8 conversion, and vice versa, with the same and other documents, on the official portable release.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by MarcoMiltenburg** [comment]
I've had this happen to me numerous times. Normally I can back out of it by not saving the file and just closing it and reopening it. But when it happened to me for about 15 minutes ago I was just to quick with hitting Ctrl-S and just lost over an hour worth of work :-(

It definitely doesn't happen always so I can imagine it being difficult to reproduce but I can confirm the report of @ThotoBot that it definitely happens from time to time.
